com.unity.cinemachine/Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -197,8 +204,11 @@ void OnEnable()
             m_BlendManager.OnEnable();
 
             s_ActiveBrains.Add(this);
+#if UNITY_EDITOR
+            m_DebugText = new DebugText(OutputCamera);
```

Comment:
This should only be created if needed.  Creating it has a performance impact because it creates and updates the panels every frame.

Reply:
Debug service should cost 0 unless it's actively being used